contentcuration/contentcuration/static/js/edit_channel/utils/number_parser.js

```diff
@@ -0,0 +1,111 @@
+/****************
```

Comment:
Regex parser for all of the different number formats (couldn't find a lightweight converter, so here is a custom implementation)